language/documentation/examples/diem-framework/move-packages/DPN/sources/DiemAccount.move

```diff
@@ -2156,6 +2157,7 @@ module DiemFramework::DiemAccount {
         }
     }
     spec epilogue_common {
+        pragma verify=false; // TODO: time out
```

Comment:
Just wondering... Is the timeout caused by our CI being weaker now?

Reply:
It timed out in my local machine, which is supposed to be better than a VM used in CI. I am not sure yet what is the exact cause for the timeout. Perhaps, it may be related to the timeout threshold setting.